app/src/app.mjs

```diff
@@ -29,7 +29,7 @@ export default async () => {
 
   // Set up Express.js
   app.set('flags', flags)
-  app.set('query parser', (query) => new URLSearchParams(query))
+  app.set('query parser', 'simple')
```

Comment:
We recently switched to `URLSearchParams` (coming to Express in future) but the community types in [`@types/express`](https://www.npmjs.com/package/@types/express) don’t expect it to be overridden. See discussion:

* https://github.com/alphagov/govuk-frontend/pull/3223#discussion_r1102469094